apps/televisit-demo/backend/createattendee/index.js

```diff
@@ -0,0 +1,77 @@
+const AWS = require('aws-sdk');
+const uuidv4 = require('uuid');
+const chime = new AWS.Chime({ region: 'us-east-1' });
+const APP_INSTANCE_ID = process.env.ChimeAppInstanceArn;
+const serverAdminUserId = 'ModeratorBot';
+const serverAdminArn = `${APP_INSTANCE_ID}/user/${serverAdminUserId}`;
+const appInstanceUserArnHeader = 'x-amz-chime-bearer';
+
+exports.handler = async event => {
+  const { userId, channel, name, meeting } = event.queryStringParameters;
+  const region = 'us-east-1';
+  console.info('Validating channel');
+  const describeChannelParams = {
+    ChannelArn: channel
+  };
+  const describeChanneRequest = await chime.describeChannel(describeChannelParams);
+  describeChanneRequest.on('build', function() {
+    describeChanneRequest.httpRequest.headers[appInstanceUserArnHeader] = serverAdminArn;
+  });
+  const describeChannelResponse = await describeChanneRequest.promise();
+  if (!describeChannelResponse.Channel) {
+    return {
+      statusCode: 404,
+      headers: {
+        'Access-Control-Allow-Headers': 'Authorization',
+        'Access-Control-Allow-Origin': '*',
```

Comment:
non-blocking for aws-samples, but here are some suggestions:

- convert the access control allow origin param to an env variable
- lock it down to the frontend accessing this backend

If you choose to continue with the wildcard, please specify this caveat in the readme so that customers are aware and can make changes if necessary.